serv/internal/authorization/opa_client_impl.go

```diff
@@ -0,0 +1,50 @@
+package authorization
+
+import (
+	"fmt"
+	"github.com/go-resty/resty/v2"
+	"github.com/pkg/errors"
+)
+
+type HasAccessRequest struct {
+	Token     string      `json:"token"`
+	Args      interface{} `json:"args"`
+	IpAddress string      `json:"ipAddress"`
+}
+
+type HasAccessResponse struct {
+	Result bool `json:"result"`
+}
+
+func NewOpaClient(config *Config) *OpaClientImpl {
+	client := resty.New()
+	client.SetBaseURL(fmt.Sprintf("http://localhost:%s/v1/data", config.Port))
+	return &OpaClientImpl{
+		client: client,
+	}
+}
+
+func (s *OpaClientImpl) HasAccess(policy string, jwt string, ipAddress string, args interface{}) (bool, error) {
+	req := HasAccessRequest{
+		Token:     jwt[7:],
```

Comment:
Can this slice fail?
```suggestion
		Token:     jwt[7:],
```